doc/source/serve/production.md

```diff
@@ -226,7 +223,7 @@ autogenerated files. The `0.0.0.0` host exposes the Serve deployments publicly,
 and `8000` is the default port for Serve's HTTP proxies. You can modify these
```

Comment:
Can we remove this paragraph? `host` and `port` aren't included in the 2.0 release.